lib/internal/abort_controller.js

```diff
@@ -35,6 +37,11 @@ function customInspect(self, obj, depth, options) {
 }
 
 class AbortSignal extends EventTarget {
+  constructor() {
+    // eslint-disable-next-line no-restricted-syntax
+    throw new TypeError('Illegal Constructor');
```

Comment:
```suggestion
    throw new TypeError('Illegal constructor');
```

Reply:
@Trott done! I also added the change to the associated test. :slightly_smiling_face: